util/__init__.py

```diff
@@ -533,3 +535,48 @@ def parse(cls, amount):
             amount = amount[1:]
         return Money(amount, currency)
 
+
+def is_string(value):
```

Comment:
Can you put this in `util/string_helpers.py`? We have some other Python compatibility functions in there.

Reply:
Moved `is_string` to `util/string_helpers.py`